modules/hello-world/pages/sample-application.adoc

```diff
@@ -1,10 +1,11 @@
 = Sample Application
+:description: The Travel Sample Application is currently available in 6 of our SDKs.
```

Comment:
Definitely not ;)
See https://github.com/couchbase/docs-sdk-java/blob/release/3.2/modules/hello-world/pages/sample-application.adoc

Reply:
Did someone say something? 🙉 

I need to merge the travel-sample docs PR for C, which will remove need for this.

Reply:
Yeah, though this will change once this is merged: https://github.com/couchbase/docs-sdk-c/pull/109/files#diff-20babd9422ff3056d572e4023a72974dc93af93f0f24170f3dc047438b801c12R2

So I will rebase and this will be gone anyways.

Reply:
haha, yep what @osfameron said 🙈

Reply:
I've merged to fix conflicts, as I caused them, but feel free to rebase locally and force push if you prefer.

Reply:
It's no problem, I expected to have those conflicts, wasn't going to merge before your change anyways. Just need fix the other bits from Richard's comments now.